src/components/input-group/daikin-input-group.ts

```diff
@@ -135,30 +148,38 @@ export class DaikinInputGroup extends LitElement {
   }
 
   override render() {
-    const inputGroupLabelClassName = cvaLabel({
-      variant: this.disabled ? "disabled" : "enabled",
-      required: this.required ? "required" : "optional",
-    });
-
-    const inputGroupHelperClassName = cvaHelper({
-      variant: this.disabled ? "disabled" : "enabled",
-    });
+    const helperType =
+      !this.disabled && !!this.error.length
+        ? "error"
+        : !!this.helper.length && !this.error.length
+          ? this.disabled
+            ? "helperDisabled"
+            : "helper"
+          : "none";
+
+    const helperText: Record<typeof helperType, string> = {
+      helper: this.helper,
+      helperDisabled: this.helper,
+      error: this.error,
+      none: "",
+    };
 
     return html`<fieldset class="content" ?disabled=${this.disabled}>
-      <label class="flex flex-col justify-center w-max gap-1 font-daikinSerif">
-        ${this.label
-          ? html`<span class=${inputGroupLabelClassName}>${this.label}</span>`
-          : null}
+      <label class="flex flex-col justify-center gap-2 w-max font-daikinSerif">
+        <span
+          class=${cvaLabel({
+            required: this.required,
+          })}
+        >
+          ${this.label}
+        </span>
```

Comment:
Maybe we should add a [sr-only](https://tailwindcss.com/docs/screen-readers) element.

```suggestion
        </span>
        ${this.required ? html`<span class="sr-only">(required)</span>` : nothing}
```